web/styles/recent_view.css

```diff
@@ -304,7 +310,10 @@
     }
 
     .recent_view_participant_overflow {
-        color: hsl(0deg 0% 0%);
+        color: var(--color-recent-view-participant-overflow-text);
+        background-color: var(
+            --color-recent-view-participant-overflow-background
+        );
```

Comment:
These can I think be done in the avatar/overflow shared block, rather than overriding it?

Reply:
Shifted these changes to avatar/overflow shared block.